src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,34 @@
+package duke.command;
+
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+/**
+ * Represents commands to be executed.
+ */
+public abstract class Command {
+    private boolean exit;
+
+    public Command(boolean exit) {
```

Comment:
I think since Command is an abstract class, we can't instantiate it, so a constructor here is kind of not needed? Do correct me if I'm missing something though!